src/DetailView/DetailView.js

```diff
@@ -191,9 +235,20 @@ class DetailView extends ConnectionComponent {
 					{this.getFilterBar()}
 					<EventTabs
 						eventTypes={eventTypes}
-						onEventTypeChange={this.handleEventTypeChange}
+						onEventTypeChange={(eventTypes) => {
+							this.resetEventChunk();
+							this.handleEventTypeChange(eventTypes);
+						}}
 						styles={[AppStyles.elementMarginTop]}/>
 					{this.getEventTable()}
+					{moreEventsAvailable && 
+						<div className={css(AppStyles.textAlignCenter, AppStyles.contentBox)}>
+							{this.state.eventChunkLoading? 
```

Comment:
Is it possible to make the ? : more visible in the code? Whitespaces on both sides perhaps would be good.